frontend/src/components/mspcontracts/BeneficiaryAddress.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
I like that this is it's own component but I feel that it's somewhat arbitrary to have the rows in the component but not the whole table. You could include it all in a component called BeneficiaryAddresses.

Then, to avoid code duplication by making the row itself a component (BeneficiaryAddress) instead of both rows together. Then you wouldn't have to reproduce the display logic. The address fields aren't nicely arranged into their own objects but you could pass into the component props as one.